machineconfiguration/v1/0000_80_machineconfigpool.crd.yaml

```diff
@@ -220,26 +217,33 @@ spec:
             status:
               description: MachineConfigPoolStatus is the status for MachineConfigPool resource.
               type: object
-              required:
-                - configuration
-                - degradedMachineCount
-                - machineCount
-                - readyMachineCount
-                - unavailableMachineCount
-                - updatedMachineCount
               properties:
+                certExpirys:
+                  description: certExpirys keeps track of important certificate expiration data
+                  type: array
+                  items:
+                    description: ceryExpiry contains the bundle name and the expiry date
+                    type: object
+                    required:
+                      - bundle
+                      - expiry
+                      - subject
+                    properties:
+                      bundle:
+                        description: bundle is the name of the bundle in which the subject certificate resides
+                        type: string
+                      expiry:
+                        description: expiry is the date after which the certificate will no longer be valid
+                        type: string
+                      subject:
+                        description: subject is the subject of the certificate
+                        type: string
                 conditions:
                   description: conditions represents the latest available observations of current state.
                   type: array
                   items:
                     description: MachineConfigPoolCondition contains condition information for an MachineConfigPool.
                     type: object
-                    required:
-                      - lastTransitionTime
-                      - message
-                      - reason
-                      - status
-                      - type
```

Comment:
this is currently not required in the MCO https://github.com/openshift/machine-config-operator/blob/40571adccaa3ef49a823abe8d7494bcdc48f2021/install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml#L293